### ISSUE TITLE
Pass aria-describedby to ComboBox on input

### ISSUE THREAD

**Issue by haneev** [jquense/react-widgets#611]
There are common cases to describe an input, currently ComboBox sets the aria-describedBy on the wrong element, this should be on the input of combox.
